src/main/java/Deadline.java

```diff
@@ -0,0 +1,25 @@
+public class Deadline extends Task {
+
+    public static final String TASK_TYPE = "D";
```

Comment:
I like the way you standardize the declaration of the constant TASK_TYPE across other subclasses of Task.

The way you name and declare the constant TASK_TYPE adheres very well to the coding standard and convention, well done!